src/helm/benchmark/presentation/summarize.py

```diff
@@ -283,28 +283,19 @@ def filter_runs_by_visibility(self, runs: List[Run], group: RunGroup) -> List[Ru
         return filtered_runs
 
     def read_runs(self):
-        """Load the corresponding runs for the run specs in run_specs.json."""
-
-        run_specs_path: str = os.path.join(self.run_suite_path, "run_specs.json")
-        if not os.path.exists(run_specs_path):
-            hlog(f"Summarizer won't run because {run_specs_path} doesn't exist yet. This is expected in a dry run.")
-            return []
-
+        """Load the runs in the run suite path."""
         self.runs: List[Run] = []
-        with open(run_specs_path) as f:
-            raw_run_specs = json.load(f)
-        for raw_run_spec in tqdm(raw_run_specs):
-            run_spec = dacite.from_dict(RunSpec, raw_run_spec)
-            run_path: str = os.path.join(self.run_suite_path, run_spec.name)
-
-            run_spec_path: str = os.path.join(run_path, "run_spec.json")
-            stats_path: str = os.path.join(run_path, "stats.json")
-
-            if os.path.exists(run_spec_path) and os.path.exists(stats_path):
-                run = self.read_run(run_path)
-                self.runs.append(run)
-            else:
-                hlog(f"WARNING: {run_path} doesn't have run_spec.json or stats.json, skipping")
+        # run_suite_path can contain subdirectories that are not runs (e.g. eval_cache, groups)
+        # so filter them out by checking if they contain ":", which only run subdirectories should have
+        run_dir_names = sorted([p for p in os.listdir(self.run_suite_path) if ":" in p])
```

Comment:
This seems a bit hacky...do all runs have `:`?  Maybe we can exclude `eval_cache` and `groups` explicitly?  Or we could just plow through everything and just slurp up everything with a `run_spec.json`...

Note that this does change the behavior because before if you remove runs from a run_spec.conf and run it, those aren't included, but now everything does.  This is perhaps only an issue when one is debugging.

it would be a bit nicer to control the set of runs that we're generating from the `run_specs.conf` rather than crawling the directory, because then we can also better control the order...  Another method is to make `helm-summarize.py` take the run specs and generate the run_specs again, but that introduces other complexity...

Reply:
Changed to exclude by name.

Also, I think the preferred dev flow is to put the debug runs in a separate suite i.e. `--suite debug`. Do we actually have a preferred order of runs?

Reply:
The runs show up in the frontend in some order on the runs page?

Reply:
Yes that's the only place that it shows up. I'm not sure that the order is particularly important?